content/Default/apps/content.json

```diff
@@ -6653,40 +6653,6 @@
     "translation_type": null,
     "tryexec": ""
   },
-  {
-    "application-id": "gnome-calculator",
-    "category": "Utility;",
-    "core": true,
-    "custom-splash-screen": "",
-    "description": "Do you ever consider how often you need to do basic or even complex math in daily life? From calculating a budget to converting measurements in a recipe to figuring out taxes, you probably use equations more regularly than you think. That\u2019s why this calculator app comes in handy. Perform simple arithmetic quickly, or allow the calculator to do more advanced work, depending on your needs. You\u2019ll never struggle to perform mathematical tasks again as long as you have this calculator app!",
-    "displayShape": "b1",
-    "exec": "gnome-calculator",
-    "featured_img": "",
-    "icon": "gnome-calculator-icon.png",
-    "is_featured": false,
-    "is_offline": false,
-    "personalities": [
-      "All"
-    ],
-    "screenshots": {
-      "C": [
-        "gnome-calculator-screenshot1.jpg"
```

Comment:
I was going to say that I thought we needed to keep these and just rename the application-id, but I was amazed to see that gnome-software gets the screenshots from flathub even though I have gnome-calculator installed from .deb packages. So, remove away!

Reply:
FWIW the distro package's screenshots seem to come from mcatanzaro:

```
wjt@camille:~$ grep image /usr/share/metainfo/org.gnome.Calculator.appdata.xml
      <image>https://people.gnome.org/~mcatanzaro/gnome-calculator-basic-mode.png</image>
      <image>https://people.gnome.org/~mcatanzaro/gnome-calculator-advanced-mode.png</image>
      <image>https://people.gnome.org/~mcatanzaro/gnome-calculator-financial-mode.png</image>
      <image>https://people.gnome.org/~mcatanzaro/gnome-calculator-programming-mode.png</image>
```

I wonder if something should be munging the built-in apps' appstreams to cache all of these on a server we control?